docs/getting_started.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Getting Started
+layout: default
+nav_order: 2
+---
+
+# Getting Started
+{: .no_toc }
+
+## Table of Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+## Installation
+
+### Local Installation
+
+Myna is a Python package which requires Python 3. You must install it locally as an
+editable package using `pip install -e .`, which is a temporary requirement that will
+be changed in future releases. The `myna` package is not on PyPI. To install `myna`
+follow the instructions below.
+
+If you have anaconda3 installed, you can use the `install_conda_env.sh`
+script to create a conda environment named "myna" that has Myna installed.
+
+```bash
+# clone repository and change directory to cloned repository
+git clone https://github.com/ORNL-MDF/Myna
+cd Myna
+
+# Myna MUST be installed as an editable package using the `-e` flag
+pip install -e .
+```
+
+Note that this does not install external, non-Python dependencies. See
+[Installation of external dependencies](#installation-of-external-dependencies)
+for additional details.
+
+#### Installation for Developers
+
+For developers, `pip install -e .[dev]` will install the optional `pytest` dependency,
+as well as `black` for autoformatting. The repository currently uses the default `black`
+formatting. Test scripts are included with the repository in the "tests" directory.
+Examples of using pytest are given below.
+
+```bash
+# Default tests for aspects myna Python package installation
+pytest
+
+# Include optional tests that check application functionality
+pytest --apps
+```
+
+#### Installation of External Dependencies
```

Comment:
Maybe put this in a separate doc since it's here and in the readme and will grow in the future

Reply:
I'm going to leave here for now, since this will probably be moving to a separate repo.